<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowsDialog</class>
 <widget class="QDialog" name="WindowsDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Windows</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource/breezEd.qrc">
    <normaloff>:/breezEd/icons/window/windows</normaloff>:/breezEd/icons/window/windows</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="windowList">
     <property name="selectionMode">
      <enum>QAbstractItemView::ExtendedSelection</enum>
     </property>
     <property name="selectionRectVisible">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="activateButton">
       <property name="text">
        <string>Activate</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/breezEd.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>233</x>
     <y>29</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>activateButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowsDialog</receiver>
   <slot>activateSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowsDialog</receiver>
   <slot>saveSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>WindowsDialog</receiver>
   <slot>closeSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>353</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>windowList</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>WindowsDialog</receiver>
   <slot>selectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>activateSelected()</slot>
  <slot>saveSelected()</slot>
  <slot>closeSelected()</slot>
  <slot>selectionChanged()</slot>
 </slots>
</ui>
